src/features/click-to-load.js

```diff
@@ -54,10 +67,17 @@ class DuckWidget {
         this.gatherDataElements()
         this.entity = entity
         this.widgetID = Math.random()
+        this.autoplay = false
```

Comment:
Why was this added?

Reply:
We were setting this later, which caused TypeScript to get upset that DuckWidget didn't have an `autoplay` property. Better to set it when constructing the Object instead, plus that keeps TypeScript happy when we set the property to true later.